jenkins_jobs/trigger_pylint_from_builds_repo.groovy

```diff
@@ -0,0 +1,38 @@
+job('trigger_pylint_from_builds_repo') {
+  label('validation_slave_label')
+  concurrentBuild()
+  properties {
+    githubProjectUrl("${DEFAULT_HOST_OS_REPOSITORIES_URL}/builds/")
+  }
+  scm {
+    git {
+      remote {
+	url("${DEFAULT_HOST_OS_REPOSITORIES_URL}/builds.git")
+	refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+      }
+      branch('$sha1')
+    }
+  }
+  triggers {
+    githubPullRequest {
+      orgWhitelist('open-power-host-os')
+      allowMembersOfWhitelistedOrgsAsAdmin()
+      cron('H/5 * * * *')
+      triggerPhrase('.*start\\W+tests.*')
```

Comment:
This triggerPhrase should be documented somewhere.  A documentation describing Jenkins and GitHub workflow is pretty much welcome.